lib/publiccloud/ssh_interactive.pm

```diff
@@ -88,7 +88,11 @@ sub ssh_interactive_leave {
         send_key 'ctrl-c';
         send_key 'ctrl-c';
         send_key 'ret';
-        last if (script_run("true") == 0);
+        # Catch exception and retry. This is necessary to catch the "command 'true' timed out" in case the serial console is busy.
+        eval {
+            last if (script_run("true") == 0);
```

Comment:
maybe we need to set also smaller waiting time in script_run ?

Reply:
also script_run has  `die_on_timeout` parameter exactly for such cases see https://github.com/os-autoinst/os-autoinst/blob/master/testapi.pm#L940

Reply:
We can. What would be a good one?

Reply:
> also script_run has `die_on_timeout` parameter exactly for such cases see https://github.com/os-autoinst/os-autoinst/blob/master/testapi.pm#L940

`With C<die_on_timeout> -1 (default) a warning will be printed to log.` - this does not explain the failures we are seeing: https://openqa.suse.de/tests/9796056#step/ssh_interactive_end/4 but we can try. Using `eval` here is anyways very ugly.

Reply:
amended and [VR](https://duck-norris.qam.suse.de/tests/11068#live) restarted